soc/posix/inf_clock/CMakeLists.txt

```diff
@@ -1,7 +1,14 @@
 # SPDX-License-Identifier: Apache-2.0
 
 zephyr_library()
+
 zephyr_library_compile_definitions(NO_POSIX_CHEATS)
+
 zephyr_library_sources(
 	soc.c
 	)
+
+zephyr_library_include_directories(
```

Comment:
is there a more central place we could put stuff like this? I see this copypasted in a lot of places. @SebastianBoe 
we want kernel/include and arch/${ARCH}/include to be in the include space for anything under arch/, board/, soc/, kernel/

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/20119#issuecomment-546308160